OJP/OJP_Trips.xsd

```diff
@@ -786,33 +796,183 @@
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="leave">
+				<xs:annotation>
+					<xs:documentation>Can be something like an elevator or a vehicle.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="enter">
+				<xs:annotation>
+					<xs:documentation>Can be something like an elevator or a vehicle.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<!-- additional enum values for individual vehicles -->
+			<xs:enumeration value="enter_roundabout">
+				<xs:annotation>
+					<xs:documentation>Entering a roundabout.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="stay_in_roundabout">
+				<xs:annotation>
+					<xs:documentation>Staying in the roundabout.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="leave_roundabout">
+				<xs:annotation>
+					<xs:documentation>Leave the roundabout.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="enter_builtup_area">
+				<xs:annotation>
+					<xs:documentation>Entering a builtup area / community.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="leave_builtup_area">
+				<xs:annotation>
+					<xs:documentation>Leave the builtup area / community.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="freewayAccessRamp">
+				<xs:annotation>
+					<xs:documentation>Access lane to highway or the like.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<!-- lane choice -->
+			<xs:enumeration value="unknown_lane_choice">
+				<xs:annotation>
+					<xs:documentation>If it is unclear which lane to choose.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="left_lane">
+				<xs:annotation>
+					<xs:documentation>If there are more than 2 lanes, then TurnAction half_left,left,sharp_left may help decide.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="middle_lane">
+				<xs:annotation>
+					<xs:documentation>If there are more than 3 lanes, then Turnaction straight defines the middle one.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="right_lane">
+				<xs:annotation>
+					<xs:documentation>If there are more than 2 lanes, then TurnAction half_right,right,sharp_right may help decide</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<!-- undefined values -->
+			<xs:enumeration value="unknown_turning_kind"/>
 		</xs:restriction>
 	</xs:simpleType>
 	<xs:simpleType name="TurnActionEnumeration">
 		<xs:annotation>
-			<xs:documentation>the range of alternative turns that can be described</xs:documentation>
+			<xs:documentation>The range of alternative turns that can be described</xs:documentation>
```

Comment:
I think possible might be more fitting?
"The range of possible turns that can be described."